server/lib/src/plugins/ecdhkeygen.rs

```diff
@@ -0,0 +1,142 @@
+use openssl::ec::{EcGroup, EcKey};
+use openssl::nid::Nid;
+use sketching::admin_error;
+
+use super::Plugin;
+use crate::event::CreateEvent;
+use crate::prelude::{Entry, EntryInvalidNew, OperationError};
+use crate::server::QueryServerWriteTransaction;
+use crate::value::PartialValue;
+use sketching::tagged_event;
+use sketching::EventTag;
+
+lazy_static! {
+    // it contains all the partialvalues used to match against an Entry's class,
+    // we just need a partialvalue to match in order to target the entry
+    static ref CLASSES_TO_UPDATE: [PartialValue; 1] = [PartialValue::new_iutf8("account")];
```

Comment:
Wrong class type. Only *humans* need to verify each other, not accounts. Accounts also include machine/service accounts.